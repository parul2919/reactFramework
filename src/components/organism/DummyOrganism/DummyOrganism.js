import React from "react";
import styled from "styled-components";
import styles from "./DummyOrganism.style";

const DummyOrganism = (props) => {
  const { className } = props;
  return (
    <section className={`dummy-organism ${className}`}>
      <div className="dummy-organism--text">
        This article i s about artistic painting. For other uses, see Painting
        (disambiguation). "Painter" redirects here. For other uses, see Painter
        (disambiguation). The Mona Lisa (1503â€“1517) by Leonardo da Vinci is one
        of the world's most recognizable paintings. An artistic depiction of a
        group of rhinos was made in the Chauvet Cave 30,000 to 32,000 years ago.
        Painting is the practice of applying paint, pigment, color or other
        medium to a solid surface (called the "matrix" or "support").[1] The
        medium is commonly applied to the base with a brush, but other
        implements, such as knives, sponges, and airbrushes, can be used. In
        art, the term painting describes both the act and the result of the
        action (the final work is called "a painting"). The support for
        paintings includes such surfaces as walls, paper, canvas, wood, glass,
        lacquer, pottery, leaf, copper and concrete, and the painting may
        incorporate multiple other materials, including sand, clay, paper,
        plaster, gold leaf, and even whole objects. Painting is an important
        form in the visual arts, bringing in elements such as drawing,
        composition, gesture (as in gestural painting), narration (as in
        narrative art), and abstraction (as in abstract art).[2] Paintings can
        be naturalistic and representational (as in still life and landscape
        painting), photographic, abstract, narrative, symbolistic (as in
        Symbolist art), emotive (as in Expressionism) or political in nature (as
        in Artivism).
      </div>
    </section>
  );
};

export default styled(DummyOrganism)`
  ${styles}
`;
