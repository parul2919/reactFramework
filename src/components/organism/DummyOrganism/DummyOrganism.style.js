import { css } from "styled-components";

export default css`
  &.dummy-organism {
    &--text {
      color: ${(props) => props.theme.colorMcdBlack};
      font-size: ${(props) => props.theme.fontS};
      padding: 10px 20px;
      text-align: justify;
      @media ${(props) => props.theme.tabletMediaQuery} {
      }
    }
  }
`;
