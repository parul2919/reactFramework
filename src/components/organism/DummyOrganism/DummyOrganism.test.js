import React from "react";
import { render } from "@testing-library/react";
import DummyOrganism from "./DummyOrganism";

describe("<Component /> spec", () => {
  it("renders the component", () => {
    const view = render(<DummyOrganism />);
    expect(view).toMatchSnapshot();
  });
});
