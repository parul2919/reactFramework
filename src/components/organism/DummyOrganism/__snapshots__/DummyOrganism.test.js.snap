// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="dummy-organism sc-bczRLJ fAGnnZ"
      >
        <div
          class="dummy-organism--text"
        >
          This article i s about artistic painting. For other uses, see Painting (disambiguation). "Painter" redirects here. For other uses, see Painter (disambiguation). The Mona Lisa (1503–1517) by Leonardo da Vinci is one of the world's most recognizable paintings. An artistic depiction of a group of rhinos was made in the Chauvet Cave 30,000 to 32,000 years ago. Painting is the practice of applying paint, pigment, color or other medium to a solid surface (called the "matrix" or "support").[1] The medium is commonly applied to the base with a brush, but other implements, such as knives, sponges, and airbrushes, can be used. In art, the term painting describes both the act and the result of the action (the final work is called "a painting"). The support for paintings includes such surfaces as walls, paper, canvas, wood, glass, lacquer, pottery, leaf, copper and concrete, and the painting may incorporate multiple other materials, including sand, clay, paper, plaster, gold leaf, and even whole objects. Painting is an important form in the visual arts, bringing in elements such as drawing, composition, gesture (as in gestural painting), narration (as in narrative art), and abstraction (as in abstract art).[2] Paintings can be naturalistic and representational (as in still life and landscape painting), photographic, abstract, narrative, symbolistic (as in Symbolist art), emotive (as in Expressionism) or political in nature (as in Artivism).
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="dummy-organism sc-bczRLJ fAGnnZ"
    >
      <div
        class="dummy-organism--text"
      >
        This article i s about artistic painting. For other uses, see Painting (disambiguation). "Painter" redirects here. For other uses, see Painter (disambiguation). The Mona Lisa (1503–1517) by Leonardo da Vinci is one of the world's most recognizable paintings. An artistic depiction of a group of rhinos was made in the Chauvet Cave 30,000 to 32,000 years ago. Painting is the practice of applying paint, pigment, color or other medium to a solid surface (called the "matrix" or "support").[1] The medium is commonly applied to the base with a brush, but other implements, such as knives, sponges, and airbrushes, can be used. In art, the term painting describes both the act and the result of the action (the final work is called "a painting"). The support for paintings includes such surfaces as walls, paper, canvas, wood, glass, lacquer, pottery, leaf, copper and concrete, and the painting may incorporate multiple other materials, including sand, clay, paper, plaster, gold leaf, and even whole objects. Painting is an important form in the visual arts, bringing in elements such as drawing, composition, gesture (as in gestural painting), narration (as in narrative art), and abstraction (as in abstract art).[2] Paintings can be naturalistic and representational (as in still life and landscape painting), photographic, abstract, narrative, symbolistic (as in Symbolist art), emotive (as in Expressionism) or political in nature (as in Artivism).
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
